frontend/src/__tests__/cypress/cypress/pages/connectionTypes.ts

```diff
@@ -179,6 +206,11 @@ class ConnectionTypesPage {
     return this;
   }
 
+  findMultiGroupSelectButton(name: string) {
+    cy.findByTestId(`select-multi-typeahead-${name}`).click();
+    return this;
+  }
```

Comment:
This should be part of the `CategorySection` class since it's all related to the category.

also Selector functions should return the selector and not `this`.